### PR TITLE
PR: Update Readme to mention that we now support PySide2.

### LINKED ISSUE #111: Update Readme for 1.3 release

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes #111

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ Copyright © 2009- The Spyder Development Team.
 **QtPy** is a small abstraction layer that lets you
 write applications using a single API call to either PyQt or PySide.
 
-It provides support for PyQt5, PyQt4 and PySide using the PyQt5 layout (where
-the QtGui module has been split into QtGui and QtWidgets).
+It provides support for PyQt5, PyQt4, PySide2 and PySide using the PyQt5 layout
```
Comment (ccordoba12) [inline README.md]: Please change here

> ... using the PyQt5 layout

to

> .. using the Qt 5 layout

**Comment by ccordoba12** [comment]
It looks good. I just added a minor comment.

File: README.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ Copyright © 2009- The Spyder Development Team.
 **QtPy** is a small abstraction layer that lets you
 write applications using a single API call to either PyQt or PySide.
 
-It provides support for PyQt5, PyQt4 and PySide using the PyQt5 layout (where
-the QtGui module has been split into QtGui and QtWidgets).
+It provides support for PyQt5, PyQt4, PySide2 and PySide using the Qt5 layout
+(where the QtGui module has been split into QtGui and QtWidgets).
 
 Basically, you write your code as if you were using PyQt5 but import Qt modules
```
Comment (goanpeca) [inline README.md]: @ccordoba12 @rlaverde we should change this to 

Basically, you write your code as if you were using PySide2 but import Qt modules from `qtpy` instead of `PySide2` (or PyQt5)

Which is what we actually do, because we use Signal instead of (`PyQt5Signal`) for instance....

Comment (ccordoba12) [inline README.md]: @rlaverde, please open a new PR for this.
